tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -262,6 +267,10 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     csinfo_.biasadd           = "BiasAdd";
     csinfo_.matmul            = "MatMul";
     csinfo_.biasaddgrad       = "BiasAddGrad";
+
```

Comment:
In line 247, we do not rewrite MklConv2D. We only rewrite BiasAddGrad into Conv2DWithBiasBackpropBias. MklConv2D is used in the example because that is the context that we look for while rewriting BiasAddGrad.